tests/agent-test/testdata/ambiguous-nix-file/ci.nix

```diff
@@ -1 +1 @@
-null
```

Comment:
Formatting test data is asking for trouble. Can we exclude `tests/agent-test/testdata`?